pom.xml

```diff
@@ -45,14 +45,14 @@
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
     <maven.compiler.source>1.8</maven.compiler.source>
     <maven.compiler.target>1.8</maven.compiler.target>
-    <surefire.version>2.22.1</surefire.version>
+    <surefire.version>3.1.2</surefire.version>
   </properties>
 
   <dependencies>
     <dependency>
       <groupId>org.apache.maven</groupId>
       <artifactId>maven-core</artifactId>
-      <version>3.8.2</version>
+      <version>3.9.4</version>
```

Comment:
It's probably not a problem - I am just not sure:

We officially support Maven 3.8.4 / 3.8.6 (https://releases.coremedia.com/cmcc-11/artifacts/CMCC%2011%20-%20Supported%20Environments.pdf).

Could there be any issues when this repo and the plugins that use this are already using artifacts of newer Maven versions than the version that is used to then execute it?

As I am unsure, I would probably stay with 3.8 in the extension plugin and here as well. Sorry, I could have said that more clearly earlier..

BUT if you feel confident that it will work I am totally fine with the update.

Reply:
Ok.  Sticking to the supported environments is a determinable policy for future updates I can live with.